src/eureka/S3_data_reduction/background.py

```diff
@@ -261,11 +261,11 @@ def fitbg(dataim, meta, mask, x1, x2, deg=1, threshold=5, isrotate=False,
                     residuals = dataslice - model
                     # Simple standard deviation (faster but prone to missing
                     # scanned background stars)
-                    # stdres = np.std(residuals)
+                    stdres = np.std(residuals)
                     # Median Absolute Deviation (slower but more robust)
                     # stdres  = np.median(np.abs(np.ediff1d(residuals)))
                     # Mean Absolute Deviation (good compromise)
-                    stdres = np.mean(np.abs(np.ediff1d(residuals)))
+                    # stdres = np.mean(np.abs(np.ediff1d(residuals)))
```

Comment:
Does this change have a postive, negative, or neutral effect on the Stage 3 background subtraction?

Reply:
It makes the BG calculation faster; the effect is neutral.